### PR TITLE
ci: use root

### PR DESCRIPTION
**Pull request by nodece**
Signed-off-by: Zixuan Liu <nodeces@gmail.com>

Fix https://github.com/streamnative/pulsarctl/pull/616

### Summary

The pulsar image default to run as the non-root user, this changed by https://github.com/apache/pulsar/pull/13376.

### CODE REVIEW

File: scripts/run-integration-tests.sh [hunk 1]
```diff
@@ -3,7 +3,7 @@ set -e
 
 readonly PROJECT_ROOT=`cd $(dirname $0)/..; pwd`
 readonly IMAGE_NAME=pulsarctl-test
-readonly PULSAR_DEFAULT_VERSION="2.9.1.2"
+readonly PULSAR_DEFAULT_VERSION="2.10.0.0-rc7"
```
Comment (EronWright) [inline scripts/run-integration-tests.sh:6]: This change seems unrelated.

**Comment by nodece** [comment]
@zymap Could you review this PR, it blocks Pulsar 2.10 release.

**Review by zymap — APPROVED** [review]
